x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/search/SearchApplicationIndexService.java

```diff
@@ -330,7 +337,7 @@ private void deleteSearchApplication(String resourceName, ActionListener<DeleteR
     }
 
     private void removeAlias(String searchAliasName, ActionListener<AcknowledgedResponse> listener) {
-        IndicesAliasesRequest aliasesRequest = new IndicesAliasesRequest().addAliasAction(
+        IndicesAliasesRequest aliasesRequest = new IndicesAliasesRequest(TimeValue.THIRTY_SECONDS, TimeValue.THIRTY_SECONDS).addAliasAction(
```

Comment:
Should we use `HARD_CODED_ENTERPRISE_SEARCH_MASTER_NODE_TIMEOUT` here as well?

Reply:
Ah yes, I missed that one somehow, thanks